server/routes/private.js

```diff
@@ -110,3 +110,48 @@ router.get('/test', async (req, res) => {
 
 module.exports = router
 
+router.post('/api/add-recurring-schedule', async (req, res) => {
```

Comment:
Hi Simran,
I would recommend moving this method above line 111 as the router should be exported WITH this method.

Reply:
Thanks Amy! I've completed this revision!